api/server.go

```diff
@@ -11,16 +12,27 @@ import (
 	"time"
 
 	"github.com/centrifuge/go-centrifuge/errors"
-
-	"github.com/centrifuge/go-centrifuge/centerrors"
+	"github.com/centrifuge/go-centrifuge/utils"
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	logging "github.com/ipfs/go-log"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/metadata"
 )
 
-var log = logging.Logger("api-server")
+// ErrNoAuthHeader used for requests when header is not passed.
+const ErrNoAuthHeader = errors.Error("'authorization' header missing")
+
+var (
+	log = logging.Logger("api-server")
+
+	// noAuthPaths holds the paths that doesn't require header to be passed.
+	noAuthPaths = [...]string{"/health.HealthCheckService/Ping"}
+
+	// TenantKey represents the key used to fetch the tenant id from context
+	TenantKey struct{}
```

Comment:
Wondering if the TenantKey should go to an `auth` package. Reason is, the ctx will be pass through to each handler, which will load the tenant info.

Reply:
ok